src/main/java/frc/robot/NoteState.java

```diff
@@ -0,0 +1,28 @@
+package frc.robot;
+
+import java.util.Arrays;
+
+public enum NoteState {
+  NO_TE(0),
+  LOADING(10),
+  CHAMBER(20),
+  READY_TO_FIRE(30),
```

Comment:
Technically, `READY_TO_FIRE` is about more than just the note; it's about the note plus the angle of the turret. I would add this as an additional queryable state or boolean.

Reply:
Resolving to merge today.  @lohengrin332 let's talk about this on Tuesday and improve forward